### PR TITLE
Adding source code to spring petclinic + additional method to configure CloudSQL

### PR DESCRIPTION
**Pull request by jabubake**
Addressing : #187 
I've also added the source code here. It is a clone of the existing [Spring PetClinic repository](https://github.com/spring-projects/spring-petclinic) with updated configuration files.

### CODE REVIEW

**Comment by jabubake** [comment]
Gentle ping

File: tutorials/run-spring-petclinic-on-app-engine-cloudsql/index.md [hunk 1]
```diff
@@ -92,15 +103,43 @@ Cloud Platform (GCP).
         spring.datasource.username=root
         spring.datasource.password=my-smart-password
 
+    See updated file [here](spring-petclinic/src/main/resources/application-mysql-datasource.properties).
+
 1.  Update `pom.xml` to include [Cloud SQL MySQL Socket Factory][socket].
     The socket library allows you to connect to your Cloud SQL instance for
-    local testing and deployment.
+    local testing and deployment. See reference `pom.xml` [here](spring-petclinic/pom-spring-datasource.xml).
```
Comment (ryanmcginnis) [inline tutorials/run-spring-petclinic-on-app-engine-cloudsql/index.md]: Change to: "Refer to [`pom-spring-datasource.xml`](spring-petclinic/pom-spring-datasource.xml)."

File: tutorials/run-spring-petclinic-on-app-engine-cloudsql/index.md [hunk 2]
```diff
@@ -136,6 +176,8 @@ https://YOUR_PROJECT_ID.appspot.com.
           - url: /.*
             script: this field is required, but ignored
 
+    Optionally, you can use the sample [app.yaml](spring-petclinic/src/main/appengine/app.yaml).
```
Comment (ryanmcginnis) [inline tutorials/run-spring-petclinic-on-app-engine-cloudsql/index.md]: Wrap app.yaml in blackticks.

File: tutorials/run-spring-petclinic-on-app-engine-cloudsql/index.md [hunk 3]
```diff
@@ -65,9 +65,12 @@ Cloud Platform (GCP).
     [HyperSQL database][hyper]. You will now switch from HyperSQL to using Cloud
     SQL as your database.
 
+1. Note: [spring-petclinic](spring-petclinic) provides a copy of the source code
+ with the required changes already completed to run Spring PetClinic using Cloud SQL.
+
 [hyper]: http://hsqldb.org/
 
-### Using Cloud SQL as your database
+### Setup Cloud SQL
```
Comment (ryanmcginnis) [inline tutorials/run-spring-petclinic-on-app-engine-cloudsql/index.md]: Change to: "Set up"

**Comment by jabubake** [comment]
@ryanmcginnis Thanks for the review, I've addressed your comments. PTAL.

**Comment by jabubake** [comment]
@labadav : Can you review ?

**Comment by jabubake** [comment]
@JimTravis : can you please assign a reviewer ?

**Comment by jmdobry** [comment]
I'll merge after @ryanmcginnis approves the PR.

**Pull request merged**